addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -119,6 +119,22 @@
                     </button>
                 </div>
             </div>
+            <div class="ms-3 mt-3 text-info additional_resource_check d-none">
+                <div t-if="widget.quiz.slideResources">
+                    <p t-if="widget.quiz.slideResourceCount > 0">Need help? Review related content:</p> 
+                    <div t-foreach="widget.quiz.slideResources" t-as="resource"
+                        t-att-data-resource-id="resource.id">
+                        <ul>
+                            <t t-if="resource.resource_type == 'url'">
```

Comment:
Seems like the condition could be part of the `t-att-href` rendering? That would significantly simplify this part.